code/src/components/album.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import '../index.css';
+import { ArtistsName } from './ArtistsName';
+import { AlbumImage } from './AlbumImage';
+
+const Album = ({ name, artists, image, link }) => {
+  console.log('album', name, artists);
+  return (
+    <section className="album">
+      <div><AlbumImage image={image} name={name} link={link} /></div>
+      <div className="textContainer">
+        <div className="albumDetails">
+          <a href={link} target="_blank" rel="noreferrer" className="albumLink">
+            <span>{name}</span>
+          </a>
+        </div>
```

Comment:
For consistency, maybe there could have been a component for ```AlbumTitle``` like it was done to the cover image, the artist name and so on? Or what was the reason for doing it this way?